module_manager.ts

```diff
@@ -247,6 +247,9 @@ export class ModuleManager {
   }
 
   static async file(spec: string) {
+    if (Deno.build.os === "windows") {
+      spec = "file://" + spec;
+    }
```

Comment:
This breaks on *nix systems.
You can import toFileUrl from `./deps.ts` and use it:

```suggestion
    spec = toFileUrl(spec).href;
```